db/migrate/20201220133400_create_volume_mappings.rb

```diff
@@ -0,0 +1,15 @@
+class CreateVolumeMappings < ActiveRecord::Migration[5.2]
+  def change
+    create_table :volume_mappings do |t|
+      t.integer :lun
```

Comment:
It is minor but can you move this below the references and unique values (so after uid_ems before timestamps) and bump the `ems` reference above the `cloud_volume` one?

This keeps the "important" stuff easy to see early in the table, it is impractical to change the layout after

Reply:
Could you see if I did it like you meant?
Also, maybe it'll help to break the fields into sections using comments?
Maybe something like below?
```
def change
# references needed by all children of EMS
t. reference :ems ...

# custom references
t.string :name...

# Fields needed by all EMS inventory items
...

# Custom fields
...

```

Reply:
Looks great @gregoryb-at-ibm, not needed for this PR but better than code comments is something that @djberg96 started doing for some tables which is adding column comments which you can actually access from psql and ActiveRecord.  This would help us when we use the schema to auto-build an openapi spec.